config/locales/en.yml

```diff
@@ -122,16 +121,65 @@ en:
       level_up_limited_description: "You're at Level %{currentLevel}, but you have targets in the Level %{minimumRequiredLevel} that have been rejected, or are pending review by a coach. You'll need to pass all milestone targets in Level %{minimumRequiredLevel} to continue leveling up."
       level_up_title: Ready to Level Up!
       level_up_description: Congratulations! You have successfully completed all milestone targets required to move to the next level in the course. Click on the "Level up" button below to continue progressing through the course.
+    CoursesCurriculum__LevelUpButton:
+      button_text_saving: Saving
+      button_text_level_up: Level Up
+    CoursesReport__Overview:
+      attempted: " Attempted"
+      average_grades: Average Grades
+      average_quiz_score: Average Quiz Score
+      targets_completed: "Completed: %{targetsCount}"
+      course_completed: Course Completed!
+      incomplete_targets: "Incomplete: %{targetsCount}"
+      level_progress: Level Progress
+      targets_pending_review: "Pending Review: %{targetsCount}"
+      personal_coaches: Personal Coaches
+      quiz: Quiz
+      schedule_coaching_session: Schedule a coaching session
+      targets_overview: Targets Overview
+    CoursesReport__SubmissionsList:
+      rejected: "Rejected"
+      completed: "Completed"
+      pending_review: "Pending Review"
+      status: Status
+      sory_by: "Sort by:"
+      filter_by_level: Filter by level
+      filter_by_level_or_status: Filter by level, or by status
+      filter_by_another_level: Filter by another level
+      filter_by_another_level_or_status: Filter by another level, or by status
+      submitted_at: Submitted At
+      submitted_on: "Submitted on %{date}"
+      submission_not_considered: This submission is not considered towards its target's completion.
+      submission_not_considered_help: This is a <span className="italic">team</span> target, and this submission is not linked to some members of your team. This can happen if a target is changed after your individual submission, to require a <span className="italic">team</span> submission, or if your team's composition changed after a <span className="italic">team</span> submission was created.
```

Comment:
@mahesh-krishnakumar Any reason `<em>` wasn't used here? 🤔

Reply:
Fixed in https://github.com/pupilfirst/pupilfirst/commit/b77e9a4f00d12f5f902801901e3277e13f1cfb2a